src/collective/embeddedpage/browser/embeddedpage.py

```diff
@@ -21,7 +26,7 @@ def get_zope_request_http_headers(self):
             if not name.startswith("HTTP_"):
                 continue
             name = name[5:].lower().replace("_", "-")
-            headers[name] = f"{value}"
```

Comment:
@wolbernd nitpicking: I am not up-to-date with latest Python 3 best practices. Aren't f-strings the way to go? cc @ericof

Reply:
@wolbernd now I also see that the imports have changed from "urllib.parse" to "six.moves.urllib_parse". Is it possible that you have some kind of Python 2 / Python 3 upgrade thingy in place? This somehow looks like the code would be compatible with Python 2 and Python 3. This would explain why it does not follow the latest Python 3 best practices. :)

Reply:
@tisto Yeah probably something like that though I'm not sure what exactly caused it since I do not have anything python2 related on my laptop...

Anyway I added a commit removing all those changes.